.devcontainer/Dockerfile

```diff
@@ -77,4 +96,4 @@ RUN echo "export HISTFILE=$HOME/commandhistory/.bash_history" >> "$HOME/.bashrc"
 
 # Install github-cli
 COPY ./.devcontainer/scripts/gh.sh /tmp/
-RUN /tmp/gh.sh
+RUN if [ "${INTERACTIVE}" = "true" ]; then /tmp/gh.sh; fi
```

Comment:
```suggestion
RUN if [ "${INTERACTIVE}" = "false" ]; then /tmp/gh.sh; fi
```

?

Reply:
We use GH cli in non-interactive mode when we destroy environments. But since that happens outside the devcontainer it means that this is not required for non-interactive hence my original change is correct.